_data/projects/BeforeIDieCode.yml

```
@@ -0,0 +1,16 @@
+name: Before I Die Code
+desc: An open source project that simplifies and guides beginners in making their first contribution to open source projects. Developers can contribute by sharing their aspirations before they die.
+site: https://github.com/BeforeIDieCode/BeforeIDieAchievements
+tags:
+  - react.js
+  - javascript
+  - community-driven
+  - css
+  - collaboration
+  - documentation
+  - beginner-friendly
+  - good-first-issue
+  - first-contribution
+upforgrabs:
+  name: help wanted
+  link: https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues
```

This should be a label or filtered list of issues within the project. Is that something. This one seems promising:

```suggestion
  link: https://github.com/BeforeIDieCode/BeforeIDieAchievements/labels/help%20wanted
```